docs/essentials/pipeline-queues.md

```diff
@@ -0,0 +1,197 @@
+
+You can use queues to prevent parallel runs of deployment pipelines that might
+cause issues in the environment you are deploying to.
+
+If your projects are deployed independently one from another, you can use the
+separate queues for each project by setting the `scope` property of the queue
+configuration to value **project**.
+
+In the following example, we are configuring the deployment pipeline that should
+be defined as a [promotion][deploying-with-promotions] to run sequentially in
+the queue called `Deployment queue`.
+
+``` yaml
+version: v1.0
+name: Production deployment
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+queue:
+  name: Deployment queue
+  scope: project
+
+blocks:
+  - name: Deploy
+    task:
+      jobs:
+      - commands:
+          - make deploy
+```
+
+In cases where multiple projects are deployed to the same environment (e.g.
+same Kubernetes cluster) you need the deployment pipelines from those projects to
+queue together. To set that up you should use the queue configuration with the
+same value for `name` and `scope` set to **organization** in all related projects.
+
+The following example illustrates the queue configuration that can be used in all
+projects that need to have a shared deployment queue.
+
+``` yaml
+version: v1.0
+name: Project A deployment
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+queue:
+  name: Shared deployment queue
+  scope: organization
+
+blocks:
+  - name: Deploy
+    task:
+      jobs:
+      - commands:
+          - make deploy
+```
+
+## Running pipelines in parallel
+
+In cases where pipeline runs are completely independent one from another, you
+might want to run them in parallel to receive faster feedback.
+
+You can configure this via `processing` property of the queue configuration.
+
+It is mostly useful for pipelines that are only running tests and when you want
+to have results for each push to GitHub, e.g. on the main branch.
+
+In case you only need the result of the pipeline from the latest push you might
+want to consider the [auto-cancel][auto-cancel] feature.  
+
+The following is the example of queue configuration that allows parallel pipelines:
+
+``` yaml
+version: v1.0
+name: Tests
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+queue:
+  processing: parallel
+
+blocks:
+  - name: Unit Tests
+    task:
+      jobs:
+      - commands:
+          - make test
+```
+
+## Conditional queue configurations
+
+In some cases it is useful to have different queue behaviors for same pipeline
+on different branches or maybe the tag-triggered pipelines should be assigned to
+a separate queue etc.
+
+Solution for these and similar cases is to have multiple queue configurations
+that are only applicable if certain conditions are met.
+
+The Semaphore supports this option trough [conditional queue configurations][cond-queue-defs-reference] as it is illustrated in the example bellow.
```

Comment:
The Semaphore → Semaphore 

Strange to break this into a new paragraph.

passive voice again... You could say (full stop) Here's an example: